simulators/portal-interop/src/main.rs

```diff
@@ -67,49 +68,144 @@ dyn_async! {
 
         // Iterate over all possible pairings of clients and run the tests (including self-pairings)
         for (client_a, client_b) in clients.iter().cartesian_product(clients.iter()) {
-            // Test block header with proof
             test.run(
                 TwoClientTestSpec {
-                    name: format!("OFFER Block Header {} --> {}", client_a.name, client_b.name),
+                    name: format!("OFFER Header block 1 {} --> {}", client_a.name, client_b.name),
```

Comment:
Just a little nitpick but perhaps you can consistently name all these it like this

```suggestion
                    name: format!("Offer Block Header: block number 1 {} --> {}", client_a.name, client_b.name),
```

Reply:
Resolved